pulp_2to3_migration/app/plugin/deb/migrator.py

```diff
@@ -35,13 +37,19 @@ class DebMigrator(Pulp2to3PluginMigrator):
     pulp2_plugin = 'deb'
     pulp2_content_models = {
         'deb': pulp2_models.DebPackage,
+        'deb_release': pulp2_models.DebRelease,
+        'deb_component': pulp2_models.DebComponent,
     }
     pulp2_collection = 'units_deb'
     pulp3_plugin = 'pulp_deb'
     pulp3_repository = pulp3_models.AptRepository
-    content_models = {
-        'deb': pulp_2to3_models.Pulp2DebPackage,
-    }
+    content_models = OrderedDict([
+        ('deb_release', pulp_2to3_models.Pulp2DebRelease),
+        ('deb', pulp_2to3_models.Pulp2DebPackage),
+        ('deb_component', pulp_2to3_models.Pulp2DebComponent),
+        ('deb_component2', pulp_2to3_models.Pulp2DebComponentPackage),
+        ('deb_component3', pulp_2to3_models.Pulp2DebReleaseArchitecture),
```

Comment:
Perhaps I need to explain a bit more what I did here:

As far as I can tell the migration plugin design more or less assumes that exactly one pulp 2 type will be mapped on exactly one pulp 2 type. However, for `pulp_deb` there are two new pulp 3 types (`ComponentPackage`, `ReleaseArchitecture`) that don't really have any equivalent pulp 2 type. The information to create these Pulp 3 types as well as (`Component`) can be extracted from the single pulp 2 `DebComponent` type (more or less).

As a result I needed several 2to3 types for only a single pulp 2 type. I am fairly confident that the keys in this dict don't actually need to be the name of the corresponding pulp 2 type so I just named them `deb_component`,  `deb_component2` and `deb_component3` here, but the entire thing is a bit workaroundish...